packages/admin/src/pages/index.page.tsx

```diff
@@ -50,4 +44,22 @@ const index = () => {
   return <div>index</div>;
 };
 
+const handleError = (error: any) => {
+  console.error('Failed to verify token', error);
+  if (error.response?.status === 401) {
```

Comment:
`error?.response?.status ` will ensure we don't get a type error if error is null or undefined